service/server_api.go

```diff
@@ -251,3 +259,26 @@ func (s *Server) registerGithub(variant string) {
 	s.configHelper.RegisterDynamicTileWithConfigVariant(github.GithubPullRequestTileType,
 		variant, &_githubModels.PullRequestParams{}, usecase)
 }
+
+func (s *Server) registerGitlab(variant string) {
+	gitlabConfig := s.config.Monitorable.Gitlab[variant]
+	// Custom UpstreamCacheExpiration only for count because gitlab has no-cache for this query and the rate limit is 30req/Hour
```

Comment:
Is this true as well?

https://docs.gitlab.com/ee/user/admin_area/settings/user_and_ip_rate_limits.html

Reply:
Indeed this is leftover from the GitHub module, I'll see to get it fixed. Another issue is that I found the global issues search api to be quite slow and hit the timeout frequently.

Reply:
According to https://gitlab.com/help/user/gitlab_com/index.md#gitlabcom-specific-rate-limits the gitlab.com rate limits are at 600 request/minute so we may not need a specific expiration.

I still need to investigate long response times from the search API.